src/test/integration/suite/init.test.it.ts

```diff
@@ -0,0 +1,53 @@
+import * as assert from "assert";
+import * as vscode from "vscode";
+import sinon from "sinon";
+import { mockGetEnvironment, mockGetOrganization } from "../mocks/server";
+import { mockGitRepoAndBranch } from "../mocks/git";
+import retry from "async-retry";
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import * as extension from "../../../../dist/extension.js";
+import { Env0EnvironmentsProvider } from "../../../env0-environments-provider";
+
+const auth = { keyId: "key-id", secret: "key-secret" };
+
+suite("init", () => {
+  test("should show environments", async () => {
+    const orgId = "org-id";
+    const envName = "my env";
+    mockGetOrganization(orgId, auth);
```

Comment:
we talked about it.  Since most of our features are basic and using our core API (which is already covered by our additional clients) it's ROI negative to setup the backend integration tests.

In addition, the integration test only supplied the vscode object, there is no real user activity, but the commands that are triggered by the user activity can be activated +-